usage-analytics/create-analytics/feedbacks_no_replied_age_repartition.sql

```diff
@@ -0,0 +1,15 @@
+WITH all_feedbacks AS (
+    SELECT DATE_TRUNC(TIMESTAMP_MILLIS(CAST(JSON_EXTRACT_SCALAR(DATA, "$.createdAt") AS INT)), DAY) AS day,count(*) feedbacks_no_response
+    FROM `<PROJECT_NAME>.firestore_export.feedback_raw_latest`
+    WHERE   JSON_EXTRACT_SCALAR(DATA, "$.status") = "pending"
+    group by day
```

Comment:
I do not understand why you need a `group by` clause here. I'd figure you'd need all feedbacks in this dataset, and then you'd compute the `period` and only then you'd group by and count how many feedbacks there are per period

Reply:
because I'm calculating the number of unanswered feedbacks for the days included in the selected period range( located at the top on the lookerstudio) and which have at least one feedback created and never answered.